apisix/utils/log-util.lua

```diff
@@ -199,7 +199,9 @@ local function get_full_log(ngx, conf)
     }
 
     if ctx.resp_body then
```

Comment:
I think we can remove this check, just check `conf.include_resp_body` is enough.